pyteal/ast/scratch_test.py

```diff
@@ -43,6 +43,19 @@ def test_scratch_load_default():
     assert actual == expected
 
 
+def test_scratch_load_index_expression():
+    expr = ScratchLoad(slot=None, index_expression=Int(1337))
+    assert expr.type_of() == TealType.anytype
+
+    expected = TealSimpleBlock([TealOp(Int(1337), Op.int, 1337)])
```

Comment:
nit: if using `TealComponent.Context.ignoreExprEquality()`, use `None` for the first argument to `TealOp` for clarity

Reply:
👍